### PR TITLE
DevWs for Zooz (The Smartest House) containing containing Zooz ZSE42 XS Water Leak Sensor

### PR DESCRIPTION
**Pull request by KevinTSH**

### CODE REVIEW

**Comment by KevinTSH** [comment]
The ZSE42 does not have the same capabilities or preferences as the ZSE41 so this DTH can't be combined with the one in PR 74779.

**Review by PKacprowiczS — CHANGES_REQUESTED** [review]
Can you add this device to existing DTH https://github.com/SmartThingsCommunity/SmartThingsPublic/blob/master/devicetypes/smartthings/zwave-water-sensor.src/zwave-water-sensor.groovy ?

**Comment by KevinTSH** [comment]
I've deleted the custom handler and added the device's fingerprint to the handler you recommended.

File: devicetypes/smartthings/zwave-water-sensor.src/zwave-water-sensor.groovy [hunk 1]
```diff
@@ -33,6 +33,7 @@ metadata {
 		fingerprint mfr: "000C", prod: "0201", model: "000A", deviceJoinName: "HomeSeer Water Leak Sensor" //HomeSeer LS100+ Water Sensor
 		//zw:Ss2 type:0701 mfr:0173 prod:4C47 model:4C44 ver:1.10 zwv:4.61 lib:03 cc:5E,55,98,9F sec:86,71,85,59,72,5A,6C,7A,84,80
 		fingerprint mfr: "0173", prod: "4C47", model: "4C44", deviceJoinName: "Leak Gopher Water Leak Sensor" //Leak Intelligence Leak Gopher Z-Wave Leak Detector
+		fingerprint mfr: "027A", prod: "7000", model: "E002", deviceJoinName: "Zooz Water Leak Sensor" //Zooz ZSE42 XS Water Leak Sensor, raw description: zw:Ss2a type:0701 mfr:027A prod:7000 model:E002 ver:1.05 zwv:7.13 lib:03 cc:5E,55,9F,6C sec:86,85,8E,59,72,5A,87,73,80,71,30,70,84,7A
```
Comment (PKacprowiczS) [inline devicetypes/smartthings/zwave-water-sensor.src/zwave-water-sensor.groovy]: Can you move raw description above the fingerprint?

**Review by PKacprowiczS — CHANGES_REQUESTED** [review]

File: devicetypes/smartthings/zwave-water-sensor.src/zwave-water-sensor.groovy (hunk 1 above)
Comment (KevinTSH) [reply devicetypes/smartthings/zwave-water-sensor.src/zwave-water-sensor.groovy:36]: Based on that other PR, is this still necessary?

Comment (KevinTSH) [reply devicetypes/smartthings/zwave-water-sensor.src/zwave-water-sensor.groovy]: I've made the requested change.

**Review by PKacprowiczS — APPROVED** [review]

**Comment by PKacprowiczS** [comment]
live 15.09

**Pull request merged**